src/Uno.UI/UI/Xaml/Controls/ContentControl/ContentControl.cs

```diff
@@ -317,6 +317,20 @@ private protected override void OnLoaded()
 			}
 		}
 
+		protected override void OnPointerPressed(PointerRoutedEventArgs args)
+		{
+			if (args.Handled)
+			{
+				base.OnPointerPressed(args);
+				return;
+			}
```

Comment:
If the `args` is already flagged as `Handled` you should not get it here. It's a bug you noticed ? If yes please open a bug and link it here, otherwise please remove.

Reply:
Not sure why I decided to add that, but yes you're right, not needed.